web/client/src/entities/Ddi/Field/RouteType.tsx

```diff
@@ -0,0 +1,58 @@
+import { ListDecorator, ScalarProperty } from '@irontec/ivoz-ui';
+import {
+    PropertyCustomFunctionComponent,
+    PropertyCustomFunctionComponentProps
+} from '@irontec/ivoz-ui/services/form/Field/CustomComponentWrapper';
+import { useStoreState } from 'store';
+import { DdiPropertyList } from '../DdiProperties';
+
+type RouteTypeValues = DdiPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<PropertyCustomFunctionComponentProps<RouteTypeValues>>;
+
+const RouteType: RouteTypeProps = (props): JSX.Element | null => {
+
+    const { _context, _columnName, property, values, formFieldFactory } = props;
+    const aboutMe = useStoreState((state) => state.clientSession.aboutMe.profile);
+
+    if (_context === 'read' || !formFieldFactory) {
+        return (
+            <ListDecorator field={_columnName} row={values} property={property} ignoreCustomComponent={true} />
+        );
+    }
+
+    const { choices, readOnly } = props;
+
+    const modifiedProperty = {...property} as ScalarProperty;
+    delete modifiedProperty.component;
+
+    const enumValues = {
+        ...modifiedProperty.enum
+    };
+
+    if (!aboutMe?.pbx) {
```

Comment:
Is ok to call it _pbx_ but I try to use _vpbx_ term everywhere

Reply:
![image](https://user-images.githubusercontent.com/800069/162919725-23abb698-f5ec-4922-ab0a-f52147badd4e.png)

Reply:
That is API response attribute name, we could replace it everywhere on it's own PR

Reply:
:+1: :see_no_evil: